.gitignore

```diff
@@ -1,5 +1,6 @@
 .env
 node_modules/
+*.exe
```

Comment:
If this is used to ignore the build output of the agent then I think instead we could use a .gitingore in the `agent` folder and just ignore the `build/out` path.